inverse/beamformer_dics.m

```diff
@@ -108,6 +108,7 @@
 projectnoise   = istrue(projectnoise);
 fixedori       = istrue(fixedori);
 dofeedback     = ~strcmp(feedback, 'none');
```

Comment:
things like dofeedback here are deprecated as feedback style. Rather we should use ft_debug, ft_info, ft_notice, ft_warning.

Reply:
hmm, you are right. I merged your latest version into this PR just earlier, and thought that I caught all of this by addressing the conflicts. Always a joy, addressing conflicts.

Reply:
dofeedback is nowhere used. also, the feedback option in the low-level functions relate to the old-style ft_progress. Let's not touch this right now, I removed the lines in the code where dofeedback was defined.